### PR TITLE
chore(predict): objective function is being deprecated

### PR DESCRIPTION
**Pull request by hussainsultan**
loss function used by XGBoost is `reg::squarederror`, formerly called `reg:linear`

### CODE REVIEW

**Review by mesejo — APPROVED** [review]
LGTM
